engine/menus/intro_menu.asm

```diff
@@ -407,8 +407,8 @@ FinishContinueFunction:
 	ld [wDontPlayMapMusicOnReload], a
 	ld [wLinkMode], a
 	ld hl, wGameTimerPaused
-	set 0, [hl]
-	res 7, [hl]
+	set GAME_TIMER_PAUSED_F, [hl]
+	res GAME_TIMER_MOBILE_F , [hl]
```

Comment:
Oh, I think this flag can be removed.. I don't think its used anywhere else not that the mobile code has been stripped. May want to verify.

Reply:
Yeah, it was only ever `set` in mobile code.